database/seeds/eggs/minecraft/egg-bungeecord.yml

```diff
@@ -0,0 +1,67 @@
+# DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO
+meta:
+  version: PTDL_v1
+exported_at: '2020-04-12T16:00:51-07:00'
+name: Bungeecord
+author: support@pterodactyl.io
+description: 'For a long time, Minecraft server owners have had a dream that encompasses a free, easy, and reliable way to connect multiple Minecraft servers together. BungeeCord is the answer to said dream. Whether you are a small server wishing to string multiple game-modes together, or the owner of the ShotBow Network, BungeeCord is the ideal solution for you. With the help of BungeeCord, you will be able to unlock your community''s full potential.'
+image: 'quay.io/pterodactyl/core:java'
+startup: 'java -Xms128M -Xmx{{SERVER_MEMORY}}M -jar {{SERVER_JARFILE}}'
+config:
+  files: |
```

Comment:
Have you verified whether the daemon can still handle that? Or does that get converted back to JSON?
If we do that as YAML from now on I also see no reason to have it as nested plain text. That would require to adapt the daemon though.

Reply:
One thing to remember is that you need to escape characters that would parse weirdly in yaml. Like these.`*&#` plus others.

Reply:
All the egg configs get transformed back into json when used in the api, but I've also verified it works with a live daemon.
I just kept the configs as strings because that's how they were in json and also they're stored as raw yaml in the database so it does make some sense to store them as plain values.

In block scalars no printable characters need to be escaped but that also comes with the downside of not being able to insert raw invisible characters.

Reply:
I feel like that was probably a design mistake on my part. This seems like a really weird way to export files, we can probably just export each of these sections as their own valid YAML that way its not a complete disaster to manually edit these files?

It parses out the same anyways.